From: SeongJae Park <sj@kernel.org>
Date: Wed, 15 Oct 2025 19:15:29 -0700
Subject: [PATCH] Docs/mm/damon/design: fix wrong link to intervals goal
 section

Commit Fixes: b243d666d107 ("Docs/admin-guide/mm/damon/usage: add
intervals_goal directory on the hierarchy") mistakenly added long
reference for intervals goal usage documentation on the design document.
Fix it.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/mm/damon/design.rst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Documentation/mm/damon/design.rst b/Documentation/mm/damon/design.rst
index b54925ea78e9..2d8d8ca1e0a3 100644
--- a/Documentation/mm/damon/design.rst
+++ b/Documentation/mm/damon/design.rst
@@ -381,8 +381,8 @@ That is, assumes 4% (20% of 20%) DAMON-observed access events ratio (source)
 to capture 64% (80% multipled by 80%) real access events (outcomes).
 
 To know how user-space can use this feature via :ref:`DAMON sysfs interface
-<sysfs_interface>`, refer to :ref:`intervals_goal <sysfs_scheme>` part of
-the documentation.
+<sysfs_interface>`, refer to :ref:`intervals_goal
+<damon_usage_sysfs_monitoring_intervals_goal>` part of the documentation.
 
 
 .. _damon_design_damos:
-- 
2.47.3

